src/main/java/de/tum/in/www1/artemis/service/connectors/localvc/LocalVCService.java

```diff
@@ -56,8 +56,12 @@ public class LocalVCService extends AbstractVersionControlService {
     @Value("${artemis.version-control.url}")
     private URL localVCBaseUrl;
 
+    private static String localVCBasePath;
```

Comment:
Why are these fields static? The fields and their usages could be made non-static simplifying injection.

Reply:
If I understand correctly, we want to make sure to keep the localVCBasePath the same across all LocalVCService instances. Though in this case it probably would not change anything removing the static, as there is only one instance of the LocalVCService (if I'm not mistaken)

Reply:
We need to access the variable without having an instance of the LocalVCService, in e.g. line 62